icu4c/source/test/intltest/loctest.cpp

```diff
@@ -3154,3 +3155,43 @@ void LocaleTest::TestBug13417VeryLongLanguageTag() {
     status.errIfFailureAndReset("\"%s\"", l.getName());
     assertEquals("equals", tag, result.c_str());
 }
+
+static const char* const tz_in_ext[] = {
+    "etadd", "tzdar", "eheai", "sttms", "arirj", "arrgl", "aruaq", "arluq",
```

Comment:
I suggest that you keep all these lists with one item per line, to make it easier to add or remove items (without ever having to reflow the line breaks).